pages/compare/ComparePage.js

```diff
@@ -118,7 +117,7 @@ export default class ResultPage extends PureComponent {
         <div className="page-container">
           <header className="result-header">
             <section className="result-header--left-section">
-              <Link href="/">
+              <Link href="/" legacyBehavior>
```

Comment:
Should we consider removing the `a` tag completely?

Reply:
We can yes, will fix

Reply:
This is fixed